hosts/dialer.go

```diff
@@ -18,6 +20,7 @@ type dialer struct {
 
 const (
 	DockerAPIVersion = "1.24"
+	K8sVersion       = "1.8"
```

Comment:
@galal-hussein @moelsayed we should also put supported docker/k8s versions to the readme; thats one of the things that has to be easy to locate

Reply:
@alena1108 Will do